www/source/docs/index.html.slim

```diff
@@ -0,0 +1,121 @@
+---
+title: Habitat documentation
+description: All of the encyclopedic knowledge of the habitat project.
+---
+section.docs--hero
+  .row
+    .columns.medium-12
+      .docs--hero--content
+        h2.docs--hero--heading Cloud Native Application Automation
+
+section.docs--intro
+  .row
+    .columns.medium-5
+      p.docs--hero--text
+        | From containers to stateful services, Habitat provides a consistent way to build and manage your applications.
+    .columns.medium-7
+      ul.docs--feature--list
+        li
+          | Habitat centers application configuration, management, and behavior around the application itself, not the infrastructure that the app runs on.
+        li
+          | This means Habitat can provide automation that can programmatically and declaratively build, run, and manage your application and services, both stateful and stateless.
+        li
+          | In addition, Habitat can be deployed and run on various infrastructure environments, such as bare metal, VM, containers, and PaaS.
+
+hr
+
+section.docs--quicklink
+  .row
+    .columns.medium-12
+      .docs--quicklink--content
+        h2 Quick Links
+  .row
+    .docs--quicklink--card--wrapper.columns.medium-4
+      .docs--quicklink--card
+        .docs--quicklink--card-heading.heading.a
+          ul.no-bullet
+            li
+              a href="https://habitat.sh/best-practices"
+                | Guide
+        .body-content
+          h4
+            a href="/docs/reference#quickstart-guide"
```

Comment:
missing

Reply:
Updated to point to recently merged web app guide